package.json

```diff
@@ -28,13 +28,17 @@
     "./package.json": "./package.json"
   },
   "dependencies": {
-    "prop-types": "^15.7.2"
+    "prop-types": "^15.7.2",
+    "@ckeditor/ckeditor5-integrations-common": "file:../ckeditor5-integrations-common"
   },
   "peerDependencies": {
     "ckeditor5": ">=42.0.0 || ^0.0.0-nightly",
+    "ckeditor5-premium-features": ">=42.0.0 || ^0.0.0-nightly",
+    "@ckeditor/ckeditor5-integrations-common": "file:../ckeditor5-integrations-common",
```

Comment:
This dependency doesn't need to be in `peerDependencies` as it's already in `dependencies`.

Reply:
yep